### DISCUSSION TITLE
Error downloading SA update

### DISCUSSION THREAD

**Discussion by gs3009** [kubilus1/autoortho#244] [General]
HI

I am trying to update the SA ortho, but I keep getting errors.

Please help

Thanks

**Comment by gs3009** [comment]
![Capture](https://github.com/kubilus1/autoortho/assets/132807056/2930b6db-bdab-43df-a8d7-cd796e6aa0cb)
Error screen

**Comment by GoofyJPR** [comment]
Hi,
Same error message for me with the update for South America (v 0.0.53) and now I no longer have the old "South America" zone (previous version)
For me the error message is for the "z_sa_00.zip", see below the end of the "autoortho.log" file:

_.../...
INFO:downloader:Split zip detected for ('C:\\Users\\webjp\\.autoortho-data\\downloads\\z_sa_15.zip',)
INFO:downloader:ZIPNAME C:\Users\webjp\.autoortho-data\downloads\z_sa_15.zip
INFO:downloader:Cleaning up parts for C:\Users\webjp\.autoortho-data\downloads\y_sa_overlays.zip
INFO:downloader:Cleaning up parts for C:\Users\webjp\.autoortho-data\downloads\z_sa_00.zip
INFO:downloader:Cleaning up parts for C:\Users\webjp\.autoortho-data\downloads\z_sa_01.zip
INFO:downloader:Cleaning up parts for C:\Users\webjp\.autoortho-data\downloads\z_sa_02.zip
INFO:downloader:Cleaning up parts for C:\Users\webjp\.autoortho-data\downloads\z_sa_03.zip
INFO:downloader:Cleaning up parts for C:\Users\webjp\.autoortho-data\downloads\z_sa_04.zip
INFO:downloader:Cleaning up parts for C:\Users\webjp\.autoortho-data\downloads\z_sa_05.zip
INFO:downloader:Cleaning up parts for C:\Users\webjp\.autoortho-data\downloads\z_sa_06.zip
INFO:downloader:Cleaning up parts for C:\Users\webjp\.autoortho-data\downloads\z_sa_07.zip
INFO:downloader:Cleaning up parts for C:\Users\webjp\.autoortho-data\downloads\z_sa_08.zip
INFO:downloader:Cleaning up parts for C:\Users\webjp\.autoortho-data\downloads\z_sa_09.zip
INFO:downloader:Cleaning up parts for C:\Users\webjp\.autoortho-data\downloads\z_sa_10.zip
INFO:downloader:Cleaning up parts for C:\Users\webjp\.autoortho-data\downloads\z_sa_11.zip
INFO:downloader:Cleaning up parts for C:\Users\webjp\.autoortho-data\downloads\z_sa_12.zip
INFO:downloader:Cleaning up parts for C:\Users\webjp\.autoortho-data\downloads\z_sa_13.zip
INFO:downloader:Cleaning up parts for C:\Users\webjp\.autoortho-data\downloads\z_sa_14.zip
INFO:downloader:Cleaning up parts for C:\Users\webjp\.autoortho-data\downloads\z_sa_15.zip
INFO:downloader:ERROR: File is not a zip file with Zipfile C:\Users\webjp\.autoortho-data\downloads\z_sa_00.zip.  Recommend retrying
INFO:downloader:Cleaning up parts for C:\Users\webjp\.autoortho-data\downloads\z_sa_00.zip_

**Comment by kubilus1** [comment]
What OS, and version of this project?

**Reply by GoofyJPR** [reply]
Windows 10.
I downloaded and installed the AutoOrtho available release on 20 June 2023. I assume it is version 0.4.x. I don't know where to find the version installed on my computer.

**Comment by gs3009** [comment]
Win 10. On both 0.4.3 and 0.4.4On 05-Jul-2023, at 20:11, kubilus1 ***@***.***> wrote:﻿
What OS, and version of this project?

**Comment by kubilus1** [comment]
As mentioned in several places, 0.4.4 or newer is *required* to update to
the latest scenery packages.  Make sure you are using a version not lower
than 0.4.4.

**Reply by GoofyJPR** [reply]
Where can I see which version of AutoOrtho is installed on my computer (Windows 10) ?
The version number does not appear in the log text file, nor in the in the user interface and there is no changelog in the installation folder.

**Reply by GoofyJPR** [reply]
In doubt, I downloaded again the latest "autoortho_win.zip" file available here:
https://kubilus1.github.io/autoortho/
Then I installed it and used it. The package update from America now works.
I do not know which version of AutoOrtho I am using, there is no information. The modification date for the files in this package is 03 July 2023.

**Comment by gs3009** [comment]
It’s not working on 0.4.4 either. I posted a screen shot with 0.4.4 error On 05-Jul-2023, at 20:47, kubilus1 ***@***.***> wrote:﻿
As mentioned in several places, 0.4.4 or newer is *required* to update to

the latest scenery packages.  Make sure you are using a version to lower

than 0.4.4.

**Comment by Shadehawke** [comment]
Can confirm that this issue is present on 0.4.4 with the North America Scenery package as well. Erroring out on na_00 and na_01

**Reply by nnabber** [reply]
I did A clean install and encountered the same problems "z_NA_00.zip is not a zipfile"

Just downloaded the latest AutoOrtho version and the problem is solved :-)

**Comment by nnabber** [comment]
maybe I was lucky then, tried it many times with the previous version

Op do 6 jul 2023 om 21:37 schreef fargo6666 ***@***.***>:

**Comment by gs3009** [comment]
Just clean up the downloads folder and delete the partly downloaded files from custom scenery. Try again It works On 07-Jul-2023, at 01:07, fargo6666 ***@***.***> wrote:﻿
I have the latest version but i still got error on both N and S america

**Reply by janjacobs66** [reply]
I deleted all in the downloads folder ( in user), nothing partially downloaded in scenery folder (x-plane)
Still can't complete NA or SA - get same error every time
"ERROR File is not a zip with Zipfile ....... .autoortho-data\downloads\"
